### ISSUE TITLE
convert onnx model of decode part of SAM failed. 

### ISSUE THREAD

**Issue by AminGeng** [alibaba/MNN#2832]
i compiled MNNConvert(2024.04.17 pulled from master branch)on ubuntu20.04. it convert onnx model of image encode part of SAM successfully. but when i tried to convert decode part, it failed. 
(both onnx models work well under onnxruntime python.)
convert command
```./MNNConvert -f ONNX --modelFile the_rest.onnx --MNNModel the_rest.mnn --bizCode MNN```
failed info:
```
The device support i8sdot:0, support fp16:0, support i8mm: 0
Start to Convert Other Model Format To MNN Model..., target version: 2.8
[16:37:20] /home/amin_u_20/work/MNN-master/tools/converter/source/onnx/onnxConverter.cpp:46: ONNX Model ir version: 8
[16:37:20] /home/amin_u_20/work/MNN-master/tools/converter/source/onnx/onnxConverter.cpp:47: ONNX Model opset version: 17
Start to Optimize the MNN Net...
Segmentation fault
```
model url: https://github.com/AminGeng/Models/blob/main/the_rest.onnx

**Comment by jxt1234** [comment]
Reproduced. Debuging.

**Comment by jxt1234** [comment]
Crash error has been fixed in 2.8.4. But still has a layernorm fuse error. Fixing.

**Comment by jxt1234** [comment]
Layernorm bug has been fixed in 2.9.0
